incubator/hnc/internal/reconcilers/hnc_config_test.go

```diff
@@ -138,12 +138,12 @@ var _ = Describe("HNCConfiguration", func() {
 		addToHNCConfig(ctx, "v1", "Secret", api.Remove)
 
 		Eventually(hasHNCConfigurationConditionWithMsg(ctx, api.MultipleConfigurationsForOneType,
-			fmt.Sprintf("APIVersion: v1, Kind: Secret, Mode: %s", api.Remove))).Should(BeTrue())
+			fmt.Sprintf("APIVersion: v1, Kind: Secret, Mode: %s", api.Remove)), statusUpdateTime).Should(BeTrue())
```

Comment:
Can you please update the comment for `statusUpdateTime` to include a reference to #871 ?

Reply:
done